client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,64 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static targets = ['total', 'label'];
+  static values = {
+    containerBody: { type: String, default: 'body' },
```

Comment:
```suggestion
    container: { type: String, default: 'body' },
```

Just call this `container` - maybe a better name is needed but this is the 'selector to find the DOM elements that contain the items we want to count'.